Keen.NET.Test/KeenClientTest.cs

```diff
@@ -46,6 +47,43 @@ public static void ResetEnv()
     }
 
     [TestFixture]
+    public class AccessKeysTest : TestBase
+    {
+        [Test]
+        public void CreateAccessKey_Success()
+        {
+            var settings = new ProjectSettingsProvider(projectId: "X", masterKey: SettingsEnv.MasterKey); // Replace X with respective value
+            var client = new KeenClient(settings);
+
+            if (UseMocks)
+                client.AccessKeys = new AccessKeysMock(settings,
```

Comment:
If the idea is just to do unit testing on the `CreateAccessKey()` method of the `KeenClient`, we can just leverage Moq instead of creating a separate class, which might be simpler.